common/STcommon.py

```diff
@@ -39,4 +75,9 @@ def parseArgs():
 	args = parseArgs()
 
 	if args.generate_keys:
-		keyGen(args.generate_keys)
\ No newline at end of file
+		keyGen(args.generate_keys)
+	if args.configure:
+		Settings.createSettings()
+		for section in ["DEFAULT","Server","Client"]:
```

Comment:
Why are we creating three categories? Personally I can't justify the need for all three. The only case where having a client and server config on one machine is if it's a dev machine, but then the listening addr is also the address being sent to. Maybe in the weird case where you have two computers each logging to each other, but at that point the LUser can rewite the settings reads IMHO.